controller/diary/exercises/getExercisesByDate.js

```diff
@@ -4,11 +4,11 @@ const getExercisesByDate = async (req, res) => {
   const { _id: owner } = req.user;
   const { date: receivedDate } = req.body;
 
-  const result = await DiaryExercise.findOne({
+  let result = await DiaryExercise.findOne({
     ownerId: owner,
     date: receivedDate,
   });
 
-  res.status(200).json(result);
+  res.status(200).json({ data: result });
```

Comment:
We don’t set the response status to 200, it’s the default

Reply:
fixed